src/components/Cluster/ClusterDetail/Ingress/__tests__/InstallIngressButton.ts

```diff
@@ -28,8 +28,11 @@ const icApp = {
 };
 
 describe('InstallIngressButton', () => {
-  it.skip('renders without crashing', () => {
-    renderWithStore(InstallIngressButton, { cluster: defaultCluster });
+  it('renders without crashing', async () => {
+    // eslint-disable-next-line @typescript-eslint/require-await
+    await act(async () => {
+      renderWithStore(InstallIngressButton, { cluster: defaultCluster });
+    });
```

Comment:
magic?

Reply:
Seems I forgot to reenable this test in #1713

Reply:
The suggested fix is just to get rid of the `... not wrapped in 'act() ...'` error